3p/twitter.js

```diff
@@ -114,7 +114,12 @@ export function cleanupTweetId_(tweetid) {
     }
   }
   // 2)
-  // Otherwise just return the first number we find.
-  // Tweet Ids are always a Int64 number.
-  return tweetid.match(/\d+/)[0];
+  // Handle malformed ids such as
+  // 585110598171631616?ref_src
+  const match = tweetid.match(/^(\d+)\?ref.*/);
```

Comment:
Any reason it's `?ref` and not `?ref_src=twsrc`

Reply:
just trying to cover more cases. don't think make a different at the end.